src/lib/error-handling.test.js

```diff
@@ -82,7 +80,7 @@ describe("error-handling", () => {
       });
 
       it("should use a default for error code", async () => {
```

Comment:
You're adding a new property, `message` to the error middleware.

Instead of changing the existing test, can you add a new test for the functionality you're adding. If you care about the order of resolution, you might want to have some extra tests as well.